css/index.css

```diff
@@ -1,3 +1,453 @@
+	margin-top: 41px;
+	display: flex;
+	flex-wrap: wrap;
+	border-bottom: 1px solid black;
+    padding-bottom: 41px;
+    width: 100%;
+}
+.service {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	padding: 20px;
+	width: 419px;
+	border: 1px solid black;
+	margin: 40px 21px 0 21px;
+	height: 252px;
+	background: #f2f2f4;
 
+}
+.service:nth-of-type(odd) {
+	margin-left: 2px;
+}
+.service:nth-of-type(even) {
+	margin-right: 0;
+}
+.service > h3 {
+	font-size: 16px;
+	margin-bottom: 17px;
+}
+.service > p {
+	line-height: 20px;
+}
+.service > a {
+	text-decoration: none;
+	color: #000;
+	font-size: 12px;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+	background: white;
+	width: 140px;
+	height: 35px;
+	padding: 10px;
+	margin-top: 16px;
+	border: 1px solid black;
+	border-radius: 5px;
 
+}
+/*CONTENT*/
+#content {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+	padding: 36px 0 33px 0;
+	margin-top: 69px;
+}
+#content p {
+	line-height: 17px;
+}
+#content .content-row {
+	display: flex;
+	justify-content: space-between;
+}
+#content .content-row section {
+	line-height: 20px;
+}
+#content .content-row section.third {
+    width: 274px;
+}
+#content .content-row section.third p {
+    line-height: 18px;
+}
+#content .content-row section.third:last-of-type {
+    margin-right: 18px;
+}
+#content .content-row section:first-of-type {
+	margin-left: 0;
+}
+#content .content-row section:last-of-type {
+	margin-right: 0;
+}
+#content #middle-img {
+	background-image: url(../img/mid-page-accent.jpg);
+	background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center;
+	display: inline-block;
+	margin: 33px 0 29px 0;
+	height: 182px;
+	width: 880px;
+}
+/*FOOTER*/
+footer {
+	display: flex;
+	flex-direction: column;
+	height: 210px;
+	margin-top: 37px;
+}
+footer address {
+	display: flex;
+	flex-direction: column;
+	height: 93px;
+	justify-content: space-between;
+}
+footer a {
+	color: black;
+	text-decoration: none;
+}
+footer #copyright {
+	display: flex;
+	font-size: 16px;
+	justify-content: center;
+	margin-top: 40px;
+}
+footer#services-footer {
+	display: flex;
+	flex-direction: column;
+	height: 500px;
+}
+footer #facts {
+	display: flex;
+	flex-direction: row;
+	height: 332px;
+	justify-content: space-between;
+}
+footer #facts > div {
+	display: flex;
+	width: 419px;
+	flex-direction: column;
+}
+footer #facts p {
+	line-height: 21px;
+}
+footer img {
+	display: block;
+	width: 416px;
+	height: 298px;
+}
+footer ul {
+	list-style: square;
+	height: 103px;
+	margin-left:20px;
+	margin-top: 25px;
+}
+footer li {
+	margin-bottom: 14px;
+}
+footer#services-footer #copyright {
+	display: flex;
+	font-size: 16px;
+	justify-content: center;
+	margin-top: 55px;
+}
+/*=========================*/
+/*===RESPONSIVE DESIGN I===*/
+/*======MEDIA QUERIES======*/
+/*=========================*/
+@media (max-width: 800px) {
+    body .container {
```

Comment:
great use of using your container to set the width for the rest of your content. 1. You don't need the body tag here and 2. Why are you having the container be smaller than the 800px? This generally means that you have to either get rid of some border and padding or that you were coding with the resizing extension, don't use that as it always adds a scrollbar. Use the chrome dev tools resizer instead.